guides/micronaut-azure-http-functions/micronaut-azure-http-functions.adoc

```diff
@@ -190,12 +196,15 @@ You can invoke the GET route:
 [source,bash]
 ----
 curl -i https://testmicronaut.azurewebsites.net/api/micronautguide
+----
+
+```
```

Comment:
I am not sure that this is proper asciidoc. I think they are Github fenced code blocks